From 77a5c1b1055719e55b9f9880df77b61ef132594e Mon Sep 17 00:00:00 2001
From: Linus Jahn <lnj@kaidan.im>
Date: Tue, 15 Oct 2019 10:16:10 +0200
Subject: [PATCH] Fix build with QXmpp 1.0.1

---
 src/MessageHandler.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/MessageHandler.cpp b/src/MessageHandler.cpp
index f347119..2746204 100644
--- a/src/MessageHandler.cpp
+++ b/src/MessageHandler.cpp
@@ -101,7 +101,7 @@ void MessageHandler::handleMessage(const QXmppMessage &msg)
 	message.setId(msg.id());
 	message.setBody(msg.body());
 	message.setMediaType(MessageType::MessageText); // default to text message without media
-#if (QXMPP_VERSION) >= QT_VERSION_CHECK(1, 0, 1)
+#if (QXMPP_VERSION) >= QT_VERSION_CHECK(1, 1, 0)
 	message.setIsSpoiler(msg.isSpoiler());
 	message.setSpoilerHint(msg.spoilerHint());
 #else
-- 
2.23.0

